robottelo/hosts.py

```diff
@@ -376,7 +376,7 @@ def install_katello_agent(self):
         if result.status != 0:
             raise ContentHostError(f'Failed to install katello-agent: {result.stdout}')
         try:
-            wait_for(lambda: self.execute('systemctl status goferd').status == 0)
+            wait_for(lambda: self.execute('service goferd status').status == 0)
```

Comment:
Why moving to older service module from systemctl ?

Reply:
For EL6 systemctl doesn't works, whereas the service command works for all